src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -168,106 +171,154 @@ The new `window-placement` permission can be queried with the
 ```js
 let granted = false;
 try {
-  const { state } = await navigator.permissions.query({ name: "window-placement" });
-  granted = state === "granted";
+  const { state } = await navigator.permissions.query({ name: 'window-placement' });
+  granted = state === 'granted';
 } catch {
   // Nothing.
 }
 ```
 
 The browser
 [can](https://webscreens.github.io/window-placement/#issue-c601b517:~:text=Permission%20check%20for%20above%3F,-partial)
-choose to show the permission prompt dynamically upon the first attempt to use any
-of the methods of the new API.
-Read on to learn more.
+choose to show the permission prompt dynamically upon the first attempt to use any of the methods of
+the new API. Read on to learn more.
 
-### The `isMultiScreen()` method
+### The `window.screen.isExtended` property
 
-To use the Multi-Screen Window Placement API, I will first call the
-`Window.isMultiScreen()` method. It returns a promise that resolves with either `true` or `false`,
-depending on whether one or multiple screens are currently connected to the machine. For my setup,
-it returns `true`.
+To find out if more than one screen is connected to my device, I access the
+`window.screen.isExtended` property. It returns `true` or `false`. For my setup, it returns `true`.
 
 ```js
-await window.isMultiScreen();
+window.screen.isExtended;
 // Returns `true` or `false`.
 ```
 
 ### The `getScreens()` method
 
-Now that I know that the current setup is multi-screen, I can obtain more
-information about the second screen using `Window.getScreens()`. It returns a promise
-that resolves with an array of `Screen` objects. On my MacBook Air 13 with a connected iPad, this
-returns an array of two `Screen` objects:
+Now that I know that the current setup is multi-screen, I can obtain more information about the
+second screen using `Window.getScreens()`. It returns a promise that resolves with an array of
```

Comment:
Is it worth mentioning that this prompts the user for permission here?

Reply:
That is covered under "The window-placement permission", but this might be an okay place to remind readers of that.

Reply:
I'll add it.